README.md

```diff
@@ -2,8 +2,18 @@
 
 All the important tests that our website should contain.
 
+## Before runnig tests
+
+-   Testers will be needed to create a test github account in order run the tests successfully.
+-   In the config folder, create a `default.json` file and store the username and password of your test github account in there. For more information about `config` read about it [here](https://www.npmjs.com/package/config).
+-   We use [puppeteer](https://pptr.dev/) and [jest](https://jestjs.io/) for testing so please familiarise yourself to these technologies if required.
 
```

Comment:
`default.json` should contain the default values, can be empty strings in this case
What you're looking for is `local.json` I suppose 🤔 

In any env, `local.json` will be given first priority. If particular values are not found in `local.json` then next priority is given to the environment specific config (`test.json` for `test` env etc) and then final priority is default.json. Therefore we should having something in default.json.

One more benefit will be people will understand the structure of config object by looking at default.json 🙂